lms/static/sass/discussion/views/_thread.scss

```diff
@@ -164,6 +164,7 @@
     .post-body {
         @extend %t-copy-sub1;
         padding: ($baseline/2) 0;
+        max-width: calc(100% - 125px);
```

Comment:
Nit: add a comment here too. I wonder if we should put some of these fixed sizes into constants so that they can be shared consistently.